### ISSUE TITLE
Clarification on RTK

### ISSUE THREAD

**Issue by stefangachter** [ChangwuLiu/InGVIO#5]
In the paper, at the end of section V, you make the following statement:
> The meter-level difference of the estimated path of GVINS [23] or InGVIO to the RTK ground truth is attributed to the inaccuracy of the atmospherical propagation models of the pseudo range.

If I am not mistaken, GVINS and InGVIO are using only pseudo-range and pseudo-range rate / Doppler shift measurements and no phase-range measurements, which would require ambiguity resolution. Did you exclude phase-range measurements from the RTK ground truth computation? If not, I would expect the meter-level difference because of considering phase-range measurements in RTK and less about the atmospherical modeling. Of course, improving atmospherical modeling will improve the result, but probably not close the gap to RTK in my view.

**Comment by ChangwuLiu** [comment]
Yes, you're right. The RTK ground truth in our fixed-wing datasets is directly acquired from the ublox receiver online, and of course carrier phase measurements are utilized. If the goal is to simply minimize the trajectorial gap, I quite agree with your suggestions to (a) cancel the atmospherical delays by differences similar to the RTK procedures rather than using more accurate prior models (b) exclude carrier phase to re-calculate RTK ground truth offline. 
Both GVINS and InGVIO calculates the error between the estimated trajectory and RTK path without any initial alignment. Imitating how the ground truth is calculated in the estimator will improve the accuracy evaluated by such `ground truth' with no doubt. It's natural to see GVINS and InGVIO perform worse than e.g. GAINS (DOI: 10.1109/ICRA46639.2022.9811362), which uses phase information since RTK is still viewed as ground truth.
However, if you visualize RTK and estimated path of InGVIO or GVINS, it's clear that though gap exists, the differences mainly lie in the vertical direction and is very stable with time evolution. A constant offset in the initial stage can almost eliminate such error from the users' perspective. Thanks a lot for your suggestions.

**Comment by stefangachter** [comment]
Thanks for the clarification. I understand your motivation and argument now. Though, I would be a little bit more precise when describing the difference. --- Note that GAINS would be a fair comparison to InGVIO if only using pseudo-range measurements and excluding phase ones. --- You know for sure better than me, but if you have a long-standing experience with GNSS, then the recent papers of "tightly coupling" GNSS with vision and IMU are somewhat "re-inventing the wheel".
